install-dev/classes/controllerHttp.php

```diff
@@ -418,22 +406,20 @@ public function getForumLink()
      */
     public function getBlogLink()
     {
-        return $this->translator->trans('http://www.prestashop.com/blog/', array(), 'Install');
+        return 'https://www.prestashop.com/blog/';
     }
 
     /**
      * Get link to support for this language
-     *
-     * Enter description here ...
      */
     public function getSupportLink()
     {
-        return $this->translator->trans('https://www.prestashop.com/en/support', array(), 'Install');
+        return 'https://www.prestashop.com/en/support';
```

Comment:
Are you sure it's a good idea. The link is different depending on the selected country :thinking: 
FR => https://www.prestashop.com/fr/support
IT => https://www.prestashop.com/it/assistenza

Reply:
Isn't the language of the page supposed to be detected and adapted automatically to the browser language or the IP localization?

Reply:
Not only, it's also about SEO

Reply:
We're talking about a dozen relevant adaptations, the rest will remain based on the English link. And it since it regards the installer, do the SEO really matter at this point? I'll ask Flavia to learn more about SEO at this step. ;-)

Reply:
You're right about the SEO :joy:!

Reply:
**UPDATE:** she replied that if Google cannot read the content, then the content won't be indexed. Also, Hugo told me that, for the URL localization to work , we need to have unlocalized URLs - I've tested it, it's better that way. So we need to remove the `/en` from the URLs. And then we're good!
🤓